### PR TITLE
Adds missing fields to ChannelAccount

### PR DESCRIPTION
**Pull request by jmnsf**
Docs: https://docs.microsoft.com/en-us/microsoftteams/platform/resources/bot-v3/bots-context

### CODE REVIEW

File: test/models/channel_acount_test.exs [hunk 1]
```diff
@@ -1,67 +1,82 @@
...
+        })
+
+      assert channel_account.id == "123456789"
+      assert channel_account.name == "Some Name"
+      assert channel_account.objectId == "a53d7526-d0e9-436f-9443-ddb317ef3608"
+      assert channel_account.givenName == "Some"
+      assert channel_account.surname == "Name"
+      assert channel_account.email == "some.name@gmail.com"
+      assert channel_account.userPrincipalName == "some.name@some-app.onmicrosoft.com"
+      assert channel_account.tenantId == "a53d7526-d0e9-436f-9443-ddb317ef3608"
```
Comment (jmnsf) [inline test/models/channel_acount_test.exs:27]: This is the only bit that really changed. Sorry about the mess but when I initially created this file I didn't have my editor properly configured and ended up pushing Windows-style endlines. This is fixed now.

**Comment by jmnsf** [comment]
Looks like the build failed while connecting to hex, should work with a retry 👍

**Comment by zabirauf** [comment]
@jmnsf Looking at the protocol, it doesn’t seem to have all those fields part of channel account. The only fields missing are `aadObjectId` and `role`

https://github.com/microsoft/botframework-sdk/blob/master/schemas/protocol/botframework.json

`

        "ChannelAccount": {
            "description": "Channel account information needed to route a message",
            "title": "ChannelAccount",
            "type": "object",
            "properties": {
                "id": {
                    "description": "Channel id for the user or bot on this channel (Example: joe@smith.com, or @joesmith or\n123456)",
                    "type": "string",
                    "title": "id"
                },
                "name": {
                    "description": "Display friendly name",
                    "type": "string",
                    "title": "name"
                },
                "aadObjectId": {
                    "description": "This account's object ID within Azure Active Directory (AAD)",
                    "type": "string",
                    "title": "aadObjectId"
                },
                "role": {
                    "description": "Role of the entity behind the account (Example: User, Bot, etc.). Possible values include:\n'user', 'bot'",
                    "type": "string",
                    "title": "role"
                }
            },
            "required": [
                "id",
                "name"
            ]
        },

`

**Comment by jmnsf** [comment]
@zabirauf maybe it's outdated? It looks like a `TeamChannelAccount` [exists](https://github.com/microsoft/botbuilder-js/blob/077a40a3900feb44226c785497b46ebda341b163/libraries/botbuilder/src/teamsActivityHandler.ts#L396-L419) that isn't in the schema, and is an [extension](https://github.com/microsoft/botbuilder-js/blob/fd7966a0eedca274e450149b190a4d8cb5f307bd/libraries/botframework-schema/src/teams/index.ts#L151-L175) of `ChannelAccount`.

There's also no `role` being returned by the API, but it's definitely returning those fields.

**Comment by jmnsf** [comment]
Rebased due to conflicts.

**Comment by jmnsf** [comment]
@zabirauf hey hey, think we can merge these PRs? 🙏

**Pull request merged**